### PR TITLE
EPMLSTRTAI-712

### PR DESCRIPTION
**Pull request by klm-1**
1. JiraSettingsDTO splitted to ...CreateDTO and ...GetDTO
2. Renamed ChangeUserRoleDTO
3. +Repackage

### CODE REVIEW

File: src/main/java/com/epam/test_generator/controllers/Admin/AdminController.java [hunk 1]
```diff
@@ -64,8 +67,8 @@ public ResponseEntity changeUserRole(@RequestBody @Valid ChangeUserRoleDTO chang
 
     @ApiOperation(value = "Get all projects", nickname = "getProjects")
     @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "OK",
-            response = ProjectDTO.class, responseContainer = "List")
+            @ApiResponse(code = 200, message = "OK",
```
Comment (pomkine) [inline src/main/java/com/epam/test_generator/controllers/Admin/AdminController.java]: What code style settings are you using?

File: src/main/java/com/epam/test_generator/controllers/Admin/response/JiraSettingsGetDTO.java [hunk 2]
```diff
@@ -17,17 +17,13 @@
     @Size(min = 1, max = 255)
     private String login;
 
-    @NotNull
-    @Size(min = 1, max = 255)
-    private String password;
-
-    public JiraSettingsDTO(String uri, String login, String password) {
+    public JiraSettingsGetDTO(Long id, String uri, String login) {
```
Comment (pomkine) [inline src/main/java/com/epam/test_generator/controllers/Admin/response/JiraSettingsGetDTO.java]: Please remove 'Get' from class name

File: src/main/java/com/epam/test_generator/services/AdminService.java [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package com.epam.test_generator.services;
 
-import com.epam.test_generator.dto.ChangeUserRoleDTO;
+import com.epam.test_generator.controllers.Admin.request.UpdateUserRoleDTO;
```
Comment (pomkine) [inline src/main/java/com/epam/test_generator/services/AdminService.java]: Avoid using upper case in package names

File: src/main/java/com/epam/test_generator/services/JiraSettingsService.java [hunk 4]
```diff
@@ -17,28 +19,25 @@
...
-                jiraSettingsDTO.getLogin(),
-                jiraSettingsDTO.getPassword());
+            JiraSettings jiraSettings = JiraSettingsTransformer.fromDTO(jiraSettingsCreateDTO);
             jiraSettingsDAO.save(jiraSettings);
             return jiraSettings;
         }
     }
 
-    public void updateJiraSettings(Long id, JiraSettingsDTO jiraSettingsDTO) {
+    public void updateJiraSettings(Long id, JiraSettingsCreateDTO jiraSettingsCreateDTO) {
```
Comment (pomkine) [inline src/main/java/com/epam/test_generator/services/JiraSettingsService.java]: It's confusing for me. Why are you using 'Create' dto in 'update' method?

File: src/main/java/com/epam/test_generator/services/AdminService.java [hunk 5]
```diff
@@ -21,26 +21,26 @@
...
 
     /**
-     * Changes user role to role specified in changeUserRoleDTO.
+     * Changes user role to role specified in updateUserRoleDTO.
      * Searching user occurs by e-mail address.
-     * @param changeUserRoleDTO
+     * @param updateUserRoleDTO
      */
-    public void changeUserRole(ChangeUserRoleDTO changeUserRoleDTO) {
+    public void changeUserRole(UpdateUserRoleDTO updateUserRoleDTO) {
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/services/AdminService.java]: Can we generify user-update operations so client would use UserUpdateDTO for all update operations?

**Pull request merged**